README.md

```diff
@@ -80,6 +80,31 @@ print(chat_history.messages)
 ```
 
 
-### Vectorstore
+### Vector Store
+
+Learn how to use the [`PGVector` vectorstore](https://github.com/langchain-ai/langchain-postgres/blob/main/examples/vectorstore.ipynb).
+
+## Google Cloud Integrations
```

Comment:
2 requests:

1. Could you add the standard test suite to the google packages and confirm that they pass the tests? It should be very quick to confirm and run the tests (~8 lines of code).

https://github.com/langchain-ai/langchain-postgres/blob/main/tests/unit_tests/test_vectorstore_standard_tests.py

2. Could you document the lack of db agnostic filtering support? it's a feature a lot of users look for (easy filtering). I think if we're recommending something from this package, users will likely expect feature parity

Reply:
1. We will have to make a few updates to our vector store ie support the method (a)get_by_ids() and support Document id fields and in metadata. 
2. I have added the note in the Readme